src/main/java/io/stargate/sgv2/jsonapi/JsonApiStartUp.java

```diff
@@ -0,0 +1,41 @@
+package io.stargate.sgv2.jsonapi;
+
+import io.quarkus.runtime.Quarkus;
+import io.quarkus.runtime.StartupEvent;
+import io.stargate.sgv2.api.common.properties.datastore.DataStoreProperties;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ApplicationScoped
+public class JsonApiStartUp {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(JsonApiStartUp.class);
+  private final DataStoreProperties dataStoreProperties;
+
+  @Inject
+  public JsonApiStartUp(DataStoreProperties dataStoreProperties) {
+    this.dataStoreProperties = dataStoreProperties;
+  }
+
+  void onStart(@Observes StartupEvent ev) {
+    LOGGER.info(
```

Comment:
SLF4J loggers have their own formatting so use of `String.format()` is not necessary. You can just use:

    LOGGER.info("VectorSearch Enabled: {}", dataStoreProperties.vectorSearchEnabled());

which avoids evaluate if logger not enabled, so bit more efficient. But more importantly bit easier to read, less to type.